tools/rosbag/src/rosbag/rosbag_main.py

```diff
@@ -218,6 +219,8 @@ def play_cmd(argv):
         parser.error("Cannot find rosbag/play executable")
     cmd = [playpath[0]]
 
+    if options.prefix:     cmd.extend(["--prefix", str(options.prefix)])
```

Comment:
Please add the new option in a consistent position in the various location. E.g. always before the `quiet` option.